### PR TITLE
Cleanup UI

### PR DESCRIPTION
**Pull request by am-aru**

### CODE REVIEW

File: components/Accordion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const FaqItem = ({ question, answer, index }: QuestionProps) => {
+  const { openFaqIndex, toggleFaq } = useFaqContext();
+  const isOpen = openFaqIndex === index;
+
+  return (
+    <>
+      <div className="subQuestion not-transform">
+        <div className="select-none flex-space-between ">
+          <button
+            onClick={() => toggleFaq(index)}
```
Comment (praveen-me) [inline components/Accordion/index.tsx]: Don't create a inline Function.
Instead use:

```
const toogleFaq = useCallback(() => toggleFaq(index), [])
```

File: components/Accordion/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            className="font-inter text-xl font-medium"
+          >
+            {question}
+          </button>
+        </div>
+        <section className="bg-white h-auto">
```
Comment (praveen-me) [inline components/Accordion/index.tsx]: Make sure to have toggle animation.

File: components/Button/Button.css
Comment (praveen-me) [inline components/Button/Button.css:1]: Why do we require this File?

File: components/ContactForm/index.tsx [hunk 3]
```diff
@@ -1,7 +1,75 @@
...
+      {options.map((option: Option) => (
+        <div key={option.value} className="flex items-center space-x-2">
+          <button
+            type="button"
+            role="radio"
+            aria-checked={selectedOption === option.value}
+            className={`options ${
+              selectedOption === option.value ? "checked" : ""
+            }`}
+            onClick={() => handleOptionChange(option.value)}
```
Comment (praveen-me) [inline components/ContactForm/index.tsx]: Don't use inline Function

File: components/ContactForm/index.tsx [hunk 4]
```diff
@@ -12,147 +80,52 @@ const ContactForm = () => {
...
-                  data-radix-collection-item
-                ></button>
-                <label className="option-text">
-                  {content.ContactForm.option4}
-                </label>
-              </div>
-            </div>
+            <label className="questions">{content.ContactForm.question}</label>
+            <DynamicOptions
+              options={[
```
Comment (praveen-me) [inline components/ContactForm/index.tsx]: Create this array dataset outside oF the component or memoised it

File: components/DecisionHelper/decisionHelper.css [hunk 5]
```diff
@@ -16,3 +16,7 @@
 .subQuestion {
   @apply w-full cursor-pointer border border-gray-200 bg-white px-12 py-10 shadow-sm;
 }
+
```
Comment (praveen-me) [inline components/DecisionHelper/decisionHelper.css:19]: Remove these unnecessary lines

File: components/Logo/index.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+function Logo({
+  children,
+  Logo,
+  logoPosition,
+  logoContainerClass,
+  logoClass,
+}: LogoProps) {
+  return (
+    <div className={clsx(logoContainerClass)}>
```
Comment (praveen-me) [inline components/Logo/index.tsx:20]: Remove children login iF unused.

File: components/ProcessFlowCard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      <span className="my-3 block font-mono text-sm text-gray-500">{text}</span>
+      <h3 className="text-xl font-semibold"> {title}</h3>
+      <p className="mt-3">
+        {description}
+
+        <br />
+        <br />
+      </p>
+
+      <Button
```
Comment (praveen-me) [inline components/ProcessFlowCard/index.tsx]: Use Something like

```
highlight && 
 <Button
        containerClass={"startButton"}
        tabIndex={0}
      >
        {"Start a Project"}
      </Button>
```

File: components/Accordion/index.tsx (hunk 1 above)
Comment (am-aru) [reply components/Accordion/index.tsx]: done

File: components/Accordion/index.tsx (hunk 2 above)
Comment (am-aru) [reply components/Accordion/index.tsx]: Done

File: components/Button/Button.css
Comment (am-aru) [reply components/Button/Button.css:1]: Deleted.

File: components/ContactForm/index.tsx (hunk 3 above)
Comment (am-aru) [reply components/ContactForm/index.tsx]: done

File: components/ProcessFlowCard/index.tsx (hunk 7 above)
Comment (am-aru) [reply components/ProcessFlowCard/index.tsx]: completed.

File: components/Logo/index.tsx (hunk 6 above)
Comment (am-aru) [reply components/Logo/index.tsx:20]: done.

**Pull request merged**